src/plugins/statusPage/public/statusPage.html

```diff
@@ -22,14 +22,16 @@ <h4 ng-if="!ui.statuses && !ui.loading" class="missing_statuses">
       No plugin status information available
     </h4>
 
-    <table class="plugin_status_breakdown row" ng-if="ui.statuses">
-      <tr>
-        <th class="col-xs-1">Name</th>
-        <th class="col-xs-11">Status</th>
+    <table class="plugin_status_breakdown .table" ng-if="ui.statuses">
```

Comment:
Oh her-dur. Yes. Interesting that it seems to be a placebo and moving the `row` is what made it right.